common/src/main/java/com/wynntils/features/trademarket/TradeMarketAutoOpenChatFeature.java

```diff
@@ -34,6 +40,9 @@ public void onChatMessageReceive(ChatMessageReceivedEvent event) {
 
         if (styledText.matches(TYPE_TO_CHAT_PATTERN)) {
             openChatWhenContainerClosed = true;
+            if (hidePrompt.get()) {
+                event.setCanceled(true);
```

Comment:
As other features rely on this chat message, you should set the event priority to lower so this doesn't cancel it before they use it

Reply:
Done, set it to LOW